src/Index.css

```diff
@@ -0,0 +1,38 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Noto-Sans", sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
```

Comment:
Just wanted to make you aware that these properties are browser/OS specific.
https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth